samples/nextjs/src/pages/_app.tsx

```diff
@@ -10,6 +11,8 @@ import NProgress from 'nprogress';
 import 'bootstrap/dist/css/bootstrap.css';
 import 'nprogress/nprogress.css';
 import 'assets/app.css';
+import { useApollo } from 'lib/GraphQLClientFactory';
+import config from 'temp/config';
```

Comment:
Put these up along with the non-css imports

Reply:
Changed